stubs/gdb/METADATA.toml

```
@@ -1,4 +1,7 @@
 version = "12.1.*"
+# This is the official web portal for GDB,
+# see https://sourceware.org/gdb/current/ for other ways of obtaining the source code.
+upstream_repository = "https://sourceware.org/git/gitweb.cgi?p=binutils-gdb.git"
```

Idk how many people are really going to be interested in the source code for `gdb` (or if this will pass our validation checks), but:

```suggestion
upstream_repository = "https://sourceware.org/git/?p=binutils-gdb.git;a=tree"
```

---

It should, it's just part of the queryparam, will try

---

is it obvious i'm not a web developer